### PR TITLE
oVirt remove release template jobs

### PR DESCRIPTION
**Pull request by Gal-Zaidman**
This is a fix for:
https://github.com/openshift/release/pull/15294

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml [hunk 1]
```diff
@@ -7014,120 +7014,6 @@ periodics:
...
-  cluster: api.ci
-  decorate: true
-  decoration_config:
-    skip_cloning: true
-  interval: 6h
-  labels:
-    job-env: ovirt
-    job-release: "4.7"
-    pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: release-openshift-ocp-installer-e2e-ovirt-4.7
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:7027]: They seem to be referenced elsewhere, e.g.:

```console
$ git grep release-openshift-ocp-installer-e2e-ovirt-4.7
ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:  name: release-openshift-ocp-installer-e2e-ovirt-4.7
core-services/template-deprecation/_allowlist.yaml:        release-openshift-ocp-installer-e2e-ovirt-4.7:
core-services/testgrid-config-generator/_allow-list.yaml:release-openshift-ocp-installer-e2e-ovirt-4.7: informing
```

**Review by petr-muller — COMMENTED** [review]
/hold

File: ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml (hunk 1 above)
Comment (Gal-Zaidman) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.7-periodics.yaml:7027]: Fixed and another problem I found :)

**Review by petr-muller — APPROVED** [review]

**Pull request merged**